doc/reference/storage_zfs.md

```diff
@@ -1,41 +1,42 @@
-(storage-zfs)=
```

Comment:
Please leave these in they are for documentation structure in MyST

see https://github.com/lxc/lxd/tree/master/doc#documentation-format